02-FirstApplicationEnvironment/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:16.04
+RUN apt-get update && apt-get install -y golang go git
+RUN git clone https://github.com/TheAnig/golang_fizzbuzz
+WORKDIR /golang_fizzbuzz
```

Comment:
I used a path right off root in my example because I was lazy, for our example this is fine, but a good best practice is to pick a directory where you will always mount and perform your actions from the host so that way it is predictable where to look when you troubleshoot inside the docker.  No changes recommended, purely informational.